src/@next/PasswordInput/PasswordInputStyle.ts

```diff
@@ -1,8 +1,12 @@
 import styled from 'styled-components';
 
 export const StyledSuffixContainer = styled.div`
-  height: 17px;
+  height: 20px;
+  transform: translateX(4px);
```

Comment:
the suffix by default has 12px padding right and 4px padding left, I can't really change that directly, so I just use transformation here... it should get be 8px padding left and 8px padding right now